addons/docs/src/frameworks/html/config.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { addParameters } from '@storybook/html';
-import { DocsPage } from '@storybook/addon-docs/blocks';
+import { DocsPage, DocsContainer } from '@storybook/addon-docs/blocks';
 
 addParameters({
+  docsContainer: DocsContainer,
```

Comment:
Still feel like we should have one parameter for docs and nest things under it :/